apps/src/p5lab/redux/animationList.js

```diff
@@ -442,18 +456,24 @@ export function appendBlankFrame() {
 }
 
 /**
- * Add an animation to the project (at the end of the list).
+ * Add an animation to the project (at the end of the list, unless a Sprite Lab project).
```

Comment:
why are we doing it differently for Sprite Lab?

Reply:
Good question! There is some institutional knowledge at play here.

When you start a new Game Lab project, there are no set starting animations, and so you just see the button to add new ones. As you add new animations, they are sorted oldest first, with the "add" button always at the end.
<img width="49%" alt="image" src="https://user-images.githubusercontent.com/43474485/218120120-c9c6e85b-0f1a-402e-855a-bd446690545d.png"><img width="49%" alt="image" src="https://user-images.githubusercontent.com/43474485/218120397-c58ea96f-3b2c-499a-b711-6c3226ff5f5b.png">
<img width="49%" alt="image" src="https://user-images.githubusercontent.com/43474485/218120512-7503a2c5-2c1b-4d1b-bbbf-f57c2f335840.png">

In Sprite Lab however, new projects start with 40+ costumes already loaded in. Using Game Lab's rules, the the hardest items to find would be the "add" button and the newest costumes selected by the student. We wanted to make the "add" button more discoverable so it was moved to the top. Similarly, students' new costumes are also sorted to the top so that it's easy for them to find them to review/edit/delete/etc.

<img width="416" alt="image" src="https://user-images.githubusercontent.com/43474485/218121336-43040bbb-6f47-4edb-b362-a0f5ed5917a5.png"><img width="394" alt="image" src="https://user-images.githubusercontent.com/43474485/218121396-fd36b762-cee8-46ff-baf5-f52b05564174.png">